### ISSUE TITLE
Project Load: ProjectTool hangs with "Object reference not set to an instance ..." when importing attached project from 2022

### ISSUE THREAD

**Issue by backYard321** [YoYoGames/GameMaker-Bugs#4940]
### Description

When opening a project that I last modified in 2022, Project Tool will state "Object reference not set to an instance of an object." and not progress any further. 

```
Core Resources : Debug Info - Exception saving resource: {
  "$GMSprite": "",
  "%Name": "background2",
  "bboxMode": 0,
  "bbox_bottom": 63,
  "bbox_left": 0,
  "bbox_right": 63,
  "bbox_top": 0,
  "collisionKind": 1,
  "collisionTolerance": 0,
  "DynamicTexturePage": false,
  "edgeFiltering": false,
  "For3D": false,
  "frames": null,
  "gridX": 0,
  "gridY": 0,
  "height": 64,
  "HTile": true,
  "layers": null,
  "name": "background2",
  "nineSlice": null,
  "origin": 0,
  "parent": {
    "name": "tilesets",
    "path": "folders/tilesets.yy",
  },
  "preMultiplyAlpha": false,
  "resourceType": "GMSprite",
  "resourceVersion": "2.0",
  "sequence": null,
  "swatchColours": null,
  "swfPrecision": 2.525,
  "textureGroupId": {
    "name": "Default",
    "path": "texturegroups/Default",
  },
  "type": 0,
  "VTile": true,
  "width": 64,
}Object reference not set to an instance of an object.
```

![image](https://github.com/YoYoGames/GameMaker-Bugs/assets/15669932/b260073a-5e8a-4a98-b62d-d4a29927e552)

[ProjecttoolLog.txt](https://github.com/YoYoGames/GameMaker-Bugs/files/14537777/ProjecttoolLog.txt)

[GMB4940_HideNSqueakSource.OG.ZIP](https://drive.google.com/file/d/1bMtrLfFtaQd7gO0JiI-i9XY64t7WpXkY/view?usp=sharing)

### Expected Change

The project should convert successfully.

### Steps To Reproduce

1. Start GameMaker 
2. Attempt to open the project
3. See the issue

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.2.0 (Monthly)

### Which operating system(s) are you seeing the problem on?

Windows 11

### Which platform(s) are you having trouble building this project for?

Windows

### Are you running GameMaker from inside your Steam library?

None

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [X] I have included a small sample project

**Comment by YYDan** [comment]
Confirmed. Added the extra logging info and also moved your sample to our private drive.

**Comment by backYard321** [comment]
> Confirmed. Added the extra logging info and also moved your sample to our private drive.

Sounds good! It's actually a game by Sokpop with the source code freely available online, so I'm not concerned about the privacy. But I appreciate the discretion!

**Comment by stuckie** [comment]
Re-opening as the automation was a little bit too quick this time.. it's not quite made a build yet.

**Comment by alicemoretti** [comment]
Verified fixed on  IDE v2024.400.0.541  Runtime v2024.400.0.560. 

Thank you.
